### DISCUSSION TITLE
Generation From IDL To python  with "-typeobject" flag results in import error.

### DISCUSSION THREAD

**Discussion by swarm59** [eProsima/Fast-DDS-Gen#113] [Show and tell]
When I generate python files form IDL using   the -typeobject flag, the test fails with the run time error show in errorLog.jpeg file.
To generate the python files I executes the following sequence of commands:
1) fastddsgen -typeobject -python Point.idl
2) cmake .
3) cmake --build .
Please note that the fastdds header are in:
/usr/local/include/fastrtps/
/usr/local/include/fastcdr/

the fastdds library are in:
/usr/local/lib
I also attach:
a) The IDL file: Point.zip
b) The Test file: testSimple.zip

Thanks for the support.
My Best,
Francesco

![errorLog](https://user-images.githubusercontent.com/10337354/166937735-090f7953-1c1e-4af6-9aeb-cead83390763.jpeg)
[Point.zip](https://github.com/eProsima/Fast-DDS/files/8631735/Point.zip)
[testSimple.zip](https://github.com/eProsima/Fast-DDS/files/8631739/testSimple.zip)

**Comment by JLBuenoLopez** [comment]
Hi @swarm59,

I have moved the discussion to the corresponding repository. I have also opened an issue with your report (#114). I will continue the conversation there.
